enatega-multivendor-admin/src/components/Restaurant/CreateRestaurant.jsx

```diff
@@ -66,7 +68,7 @@ const CreateRestaurant = props => {
   const [minimumOrderError, setMinimumOrderError] = useState(null)
   const [salesTaxError, setSalesTaxError] = useState(null)
   const [restaurantCuisines, setRestaurantCuisines] = React.useState([])
-
+  const [isOpen, setIsOpen] = useState(true); // State to control visibility
```

Comment:
Instead of creating state in this component, call onClose function from prop, it will close the popup, right the the popup is rendered but the content is empty.

Reply:
Resolved